services/rebuild-app.js

```diff
@@ -5,7 +5,7 @@
 const { exec } = require('child_process');
 
 module.exports = () => new Promise((resolve, reject) => {
-  const buildProcess = exec('npm run build'); // Trigger the build command
+  const buildProcess = exec('NODE_OPTIONS="--max-old-space-size=512" npm run build'); // Trigger the build command
```

Comment:
I think the default is actually 512 (on 32-bit systems) and 1760 on all other systems. This number might need to be a bit higher if we consistently see mem issues

Reply:
Totally agree.